.github/workflows/scorecards.yml

```diff
@@ -0,0 +1,69 @@
+name: Scorecard supply-chain security
+on:
+  # For Branch-Protection check. Only the default branch is supported. See
+  # https://github.com/ossf/scorecard/blob/main/docs/checks.md#branch-protection
+  branch_protection_rule:
+  # To guarantee Maintained check is occasionally updated. See
+  # https://github.com/ossf/scorecard/blob/main/docs/checks.md#maintained
+  schedule:
```

Comment:
I'd suggest removing this block, for two reasons:

1. We don't need to run the job on every push to main. 
2. The job is scheduled to run at 4:27 PM (16:27) every Monday, that's a long period.

I'd refactor to something similar to https://github.com/keycloak/keycloak/blob/main/.github/workflows/snyk-analysis.yml. Add the:

```
on: 
  workflow_dispatch:
```
And include it as part of the nightly jobs here:

https://github.com/keycloak/keycloak/blob/main/.github/workflows/schedule-nightly.yml#L31-L37

Does it make sense?